scripts/SetupServiceFabric/SetupServiceFabric.sh

```diff
@@ -5,6 +5,11 @@
 # It also sets up Azure Service Fabric CLI
 #
 # Usage: sudo ./SetupServiceFabric.sh
+# Setting up Service Fabric from local .deb packages is supported for WSL2 based Linux VM
```

Comment:
You should rephrase this to say the runtime and sdk are required when this installation is supposed to happen from those. This can be either of the case: WSL or real Linux system.

If this is a WSL installation or real linux system, only that check should drive if you do genie related things. Rest of the things should remain as is. please avoid copy pasting same things again and avoid duplication.

Reply:
Additional code of block is introduced for - 
1. Installation of systemd-genie when systemd is not running as PID1, this block will setup a genie namespace.
2. Installation of sf sdk from local packages if path is provided in the script, it is supported in both WSL2 distribution and standard linux VM

For the rest of the code, re-usability has been ensured, however to ensure this genieCommand prefix is introduced which will be "genie -c" to fire the installation inside genie namespace, for standard VM is will be an empty string making the command equivalent to previous commands.